### PR TITLE
make sure etag is a string for GET room_keys/version

### PR DESCRIPTION
**Pull request by uhoreg**
Thanks to sorunome for noticing.

The spec says that `etag` should be a string, but it's stored in the DB as an int, so we need to convert it before we return it.

### CODE REVIEW

**Comment by uhoreg** [comment]
Uhm...  Someone broke CI...

/me throws it in the review queue and slowly walks away

File: changelog.d/7691.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Fix an incorrect type in the `GET room_keys/version` endpoint.
```
Comment (clokep) [inline changelog.d/7691.bugfix:1]: We've been trying to include info about when a bug was introduced in the changelogs -- or has this just always been this way?

Comment (uhoreg) [reply changelog.d/7691.bugfix:1]: It's pretty much always been that way (ever since that field was included in the result)

**Comment by clokep** [comment]
Note that the broken build is from a busted virtualenv release. Seems like they're fixing it.

File: synapse/handlers/e2e_room_keys.py [hunk 2]
```diff
@@ -351,6 +351,7 @@ def get_version_info(self, user_id, version=None):
                     raise
 
             res["count"] = yield self.store.count_e2e_room_keys(user_id, res["version"])
+            res["etag"] = str(res["etag"])
```
Comment (clokep) [inline synapse/handlers/e2e_room_keys.py:354]: I checked briefly on Friday and this matches the other places we call `get_e2e_room_keys_version_info`.

File: changelog.d/7691.bugfix (hunk 1 above)
Comment (clokep) [reply changelog.d/7691.bugfix:1]:
```suggestion
Fix a long standing bug where the response to `GET room_keys/version` endpoint had the incorrect type for the `etag` field.
```

**Pull request merged**